### ISSUE TITLE
Design a pop-up banner for events

### ISSUE THREAD

**Issue by jenlampton** [backdrop-ops/backdropcms.org#424]
We have a few events coming up where it might be useful to have a Backdrop CMS pop-up banner. BADCamp 2017, and Collision Conf 2018. Does anyone want to work with us to create a masterful piece of advertising material? @dariusgarza ?

**Comment by laryn** [comment]
@jenlampton Can you post the specs here?

**Comment by jenlampton** [comment]
I don't know the specs for a pop-up banner, but I expect there are some
standard sizes?

**Comment by laryn** [comment]
What do you want on it? Just the name and logo?

**Comment by jenlampton** [comment]
Maybe a tagline? Maybe some bullets? Whatever makes sense!

**Comment by laryn** [comment]
Not in final form obviously, but here's a rough version for comments, incorporating a digital painting (also somewhat in progress) which I've been working on when I need a break from code. 

![backdrop-banner-01](https://user-images.githubusercontent.com/871421/68426910-c7e62300-016e-11ea-9bfb-ef29764fb870.jpg)

**Comment by klonos** [comment]
Love it ❤️ 👍

**Comment by laryn** [comment]
Here's some draft text for anyone who wants to do some wordsmithing:

> **Drupal 7...evolved**

> The power and flexibility that you’ve come to love from Drupal 7 has been combined with a ton of usability improvements and impressive new features, including configuration management, the ability to update core with a few clicks, and a system for building and assigning layouts to pages or entire sections of your website. All this while maintaining a commitment to affordability and placing a high value on backwards compatibility.

**Comment by klonos** [comment]
Long sentences (especially without proper/frequent punctuation) make things hard to read/understand for some people ...especially those whose primary language is not English. Here's my take:

> **Drupal 7...evolved**
>
> The power and flexibility that you’ve come to love from Drupal 7 has been combined with a ton of usability improvements, and impressive new features. These include:
> - Configuration Management
> - The ability to update core with a few clicks
> - a system for building and assigning layouts to pages or entire sections of your website
> - ???
> All that, while maintaining a commitment to affordability, and placing a high value on backwards compatibility.

**Comment by klonos** [comment]
...I think that along with "power" and "flexibility", we should also mention "simplicity", or "lower barrier to entry", or something like that.

**Comment by dariusgarza** [comment]
Since people are already talking about D9, does it make sense to frame
backdrop in terms of D7? Kinda makes the platform seem outdated.

I also think you can drop all of the paragraph text. A tagline is more than
enough and nobody reads. Should be more of a billboard than a brochure.

**Comment by klonos** [comment]
> Since people are already talking about D9, does it make sense to frame
backdrop in terms of D7? Kinda makes the platform seem outdated.

I think that people that are onboard the D8 train will find it outdated. These are the ones looking at D9, and these are currently 300k vs the 700k+ still on D7: https://www.drupal.org/project/usage/drupal ...our goal has always been to attract those that are looking for a solution past D7 that is not D8 (and now D9). IOW, I think that we'll be OK with mentioning D7 🙂

**Comment by olafgrabienski** [comment]
> Since people are already talking about D9, does it make sense to frame backdrop in terms of D7?

Do we want to mention Drupal at all (in the tagline)? I'd prefer something like the slogan on the homepage of backdropcms.org:

*The free and Open Source CMS that helps you build websites for businesses and non-profits*

**Comment by dariusgarza** [comment]
Great point. Why is the messaging so Drupal forward? If anything, it might
scare people away. Drupal is so not on trend atm.

**Comment by stpaultim** [comment]
I think that the importance of mentioning Drupal depends upon the audience. At Drupal events, our audience is Drupal 7 users that are looking for an alternative to Drupal 8/9. For them, mentioning Drupal is really important. 

If we are using a banner like this at a non-Drupal event, then there may be little need to emphasize Drupal at all. Can we appeal to users at both kinds of events with one banner?

**Comment by serundeputy** [comment]
re; drupal 
I say leave it out, and then Drupal will naturally come up in the human conversation with plenty of room for context and explanation.

keep the banner simple in terms of words.

**Comment by jackaponte** [comment]
I agree with @stpaultim's take on the importance of mentioning Drupal depending on the audience. That said, if we're going to get just _one_ banner, I'd go with leaving Drupal off the banner--and including very few words, kinda like what @olafgrabienski suggested above--and figuring out other ways to communicate the connection the Drupal both visually when that's desired, for people just table browsing and not stopping to talk or pick up materials.

I also agree with Tim about the colorful dragon with a "real life" looking background seeming a bit too complex for a banner display; I'd be more inclined to stay more on-brand and incorporate Drop the Dragon into a more simple rendering of a dragon coming out of an egg. I'd support making the egg Drupal blue for some subtle shade, heh.

**Comment by BWPanda** [comment]
My personal constructive criticism of the proposed banner design and thoughts in general for an event banner:

- Keep the background white with black text/logo (this *may* save on printing costs, not 100% sure)
- The digital painting is very nice and detailed, kudos to @laryn for that! But I don't think it works on an event banner: it's not the standard Backdrop mascot (e.g. marketing/branding consistency), and it's too 'busy' (I think it wouldn't be obvious from a distance what it is). Just our mascot 'Drop' should be fine.
- I think text should be a sentence maximum, and maybe bullet points of just a few words each. E.g. the slogan that @olafgrabienski suggested and maybe bullet points of the main points from the front-page of our website
- Keep the 'backdropcms.org' web address, but below it put just logos of where else to find us (e.g. Twitter, Facebook, GitHub, YouTube, etc.)

**Comment by laryn** [comment]
Here's a quick mockup incorporating many of the more "on brand" suggestions being made, for further discussion.

![backdrop-banner-01](https://user-images.githubusercontent.com/871421/69087765-79f1da80-0a0c-11ea-9ca3-7a9fd701f83d.png)

**Comment by jenlampton** [comment]
I love both! I love the "Evolved" language and imagery, but agree it might not be on-brand. I like this version too, but wonder if we need all the bullets? Are some of these more important than others?

**Comment by jackaponte** [comment]
Agreed with @jenlampton on all of the above! I really like the new version of the banner, thank you @laryn. I agree that it could use less text (and maybe less periods!)

Perhaps something like this instead for the bullet list?

* Fast sites on minimal hosting resources
* Responsive & mobile-ready, out-of-the-box
* Easily customizable & extensible
* End-user focused, site-builder & developer-friendly
* Secure by design

I struggled most with "mobile-ready" which is kinda random. "Mobile-friendly" seems a bit weak, and I'm not sure whether "mobile-first" is really accurate (how does one judge that?)

**Comment by jenlampton** [comment]
I like `mobile-ready`!

**Comment by laryn** [comment]
I added @jackaponte's list, bolded some keywords, and also added a slight grey background stripe in the style of the website theme.

![backdrop-banner-01](https://user-images.githubusercontent.com/871421/69273521-21eadd80-0b9e-11ea-94eb-c1557a32d046.png)

**Comment by BWPanda** [comment]
Maybe a tiny bit of whitespace between each bullet point...? (getting picky now :wink: )

**Comment by laryn** [comment]
Here's a PDF of the current iteration (with a little space between each point):
[backdrop-banner.pdf](https://github.com/backdrop-ops/backdropcms.org/files/3928235/backdrop-banner.pdf)

**Comment by jenlampton** [comment]
Visual :) 
<img width="427" alt="Screen Shot 2019-12-05 at 12 04 30 PM" src="https://user-images.githubusercontent.com/397895/70269535-72923700-1757-11ea-8078-1b3d17348967.png">

**Comment by klonos** [comment]
I really like these designs ❤️ ...but re wording, I think that we should perhaps include mentions of Drupal 7 ...at least till 2021. Reason being that till then we need to grab the attention of people still in D7, whereas the latest design makes it sound like a separate product (with which D7 people may be unfamiliar - so we lose the whole "the D7 you know, but better" argument).

**Comment by stpaultim** [comment]
Did we create a banner yet. Marking good first issue to encourage new folks to comment on this and provide additional input, until we actually produce the banner. This was priority, back when we were attending conferences, but right now it has dropped pretty far down the list of things to do.
